apiserver/pkg/chat/chat.go

```diff
@@ -46,24 +47,24 @@ func AppRun(ctx context.Context, req ChatReqBody, respStream chan string) (*Chat
 	token := auth.ForOIDCToken(ctx)
 	c, err := client.GetClient(token)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("failed to get a dynamic client: %w", err)
 	}
 	obj, err := c.Resource(schema.GroupVersionResource{Group: v1alpha1.GroupVersion.Group, Version: v1alpha1.GroupVersion.Version, Resource: "applications"}).
 		Namespace(req.AppNamespace).Get(ctx, req.APPName, metav1.GetOptions{})
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("failed to get application: %w", err)
 	}
 	app := &v1alpha1.Application{}
 	err = runtime.DefaultUnstructuredConverter.FromUnstructured(obj.UnstructuredContent(), app)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("failed to convert application: %w", err)
```

Comment:
How about that we all use this `github.com/pkg/errors`

```golang
errors.wrap(err,"Failed to convert application")
```

Reply:
I'm good if we have a better choice on errors wrapper.

Reply:
Just make sure that we wrap errors in a common way.

Reply:
> How about that we all use this `github.com/pkg/errors`

`github.com/pkg/errors` has been archived in Dec 1, 2021. https://github.com/pkg/errors/issues/245

IMO, use `pkg/errors` `wrap` function, we also need `Cause` function for recursively retrieve error, but we do not use it. 

So, it seems that we don't care about the wrong stack, so we can just use the `%w` syntax of the standard library is enough.